stixpy/net/client.py

```diff
@@ -223,7 +223,6 @@ def register_values(cls):
             attrs.Level: [
                 ("L0", "STIX: commutated, uncompressed, uncalibrated data."),
                 ("L1", "STIX: Engineering and UTC time conversion ."),
-                ("ANC", "STIX: Ancillary Data like aspect."),
                 ("L2", "STIX: Calibrated data."),
                 ("ANC", "STIX: Ancillary data."),
```

Comment:
@nicHoch I think the `ANC` level attr was in twice so I removed it but that seem to change a few of the test by 1 which I have assumed was an extra entry for the second `ANC`